munininfluxdb/commands/fetch.py

```diff
@@ -1,31 +1,22 @@
 #!/usr/bin/env python
 from __future__ import print_function
-import pwd
 import json
 import os
 import sys
-import argparse
 from collections import defaultdict
 
 from munininfluxdb.utils import Symbol
 from munininfluxdb.settings import Defaults
 
 import influxdb
+import storable
 
-try:
```

Comment:
Before touching `munin-influxdb` I also sent a PR for `storable` which makes the module available on pypi. This has been accepted and we can install it via setup.py dependencies. The `vendor` folder is no longer needed.